test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -645,7 +651,12 @@ TEST_P(ExtProcIntegrationTest, GetAndSetTrailersIncorrectlyOnResponse) {
     return true;
   });
 
-  verifyDownstreamResponse(*response, 500);
+  if (Runtime::runtimeFeatureEnabled(Runtime::defer_processing_backedup_streams)) {
+    // We get a reset since we've received some of the response already.
+    ASSERT_TRUE(response->waitForReset());
+  } else {
+    verifyDownstreamResponse(*response, 500);
+  }
```

Comment:
For my understanding how does this change with deferred? Are we holding the response before sending it with !deferred?